kimchi/src/circuits/witness/mod.rs

```diff
@@ -4,6 +4,7 @@ mod copy_bits_cell;
 mod copy_cell;
 mod copy_shift_cell;
 mod crumb_cell;
```

Comment:
This is a misnomer and should be renamed.  A crumb is 2 bits and a nybble is 4.  Maybe this is more acurately `PrefixBitsCell` or something like that?

Reply:
oh, forgot the nybble name